server/service/src/invoice_line/inbound_shipment_line/insert/mod.rs

```diff
@@ -24,7 +24,6 @@ pub struct InsertInboundShipmentLine {
     pub expiry_date: Option<NaiveDate>,
     pub number_of_packs: f64,
     pub total_before_tax: Option<f64>,
-    pub tax: Option<f64>,
```

Comment:
I think at least on the service layer we can keep it possible to set the line tax directly. We can still decide if we want to expose this through the api (can't think we shouldn't and defer the restriction to the UI).